initiatives/submit-a-request/index.html

```diff
@@ -0,0 +1,433 @@
+{% set initiative = get_document("initiative", "submit-a-request") %}
+{% set path = "/initiatives/" + initiative.slug %}
+
+{% extends "layout-side-nav.html" %}
+{% import "_vars-initiatives.html" as vars with context %}
+{% set active_nav_id = initiative.slug %}
+
+{% set activity_type = "FOIA" %}
+{% from "post-macros.html" import activity_snippets as activity_snippets with context %}
+{% set activities_feed = activity_snippets(activity_type) %}
+
+{% block title -%}
+    {{ initiative.title | safe if initiative.title }}
+{%- endblock %}
+
+{% block content_main_modifiers -%}
+    {{ super() }} content__flush-bottom
+{%- endblock %}
+
+{% block content_main %}
+
+    {% if initiative.title %}
+        <h1>{{ initiative.title | safe }}</h1>
+    {% endif %}
+
+    {% if initiative.content %}
+    <section class="initiative_description block block__flush-top">
+        {{ initiative.content | safe }}
+
+        <!-- Extra Content. Remove once content updates. -->
```

Comment:
Prefer jinja comments `{# #}`, unless it's important the comment show up in the browser.